x-pack/plugins/uptime/public/components/monitor/synthetics/__tests__/step_screenshot_display.test.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { shallowWithIntl, mountWithIntl } from 'test_utils/enzyme_helpers';
+import React from 'react';
+import * as reactUse from 'react-use';
+import { StepScreenshotDisplay } from '../step_screenshot_display';
+
+describe('StepScreenshotDisplayProps', () => {
+  // @ts-ignore missing fields don't matter in this test, the component in question only relies on `isIntersecting`
+  jest.spyOn(reactUse, 'useIntersection').mockImplementation(() => ({
+    isIntersecting: true,
+  }));
+
+  it('displays screenshot thumbnail when present', () => {
+    // reactUseSpy.
```

Comment:
Delete this line?

Reply:
Same as the console log, happened due to a blanket commit that I forgot to undo.